tools/c7n_mailer/tests/default.html.j2

```diff
@@ -0,0 +1,281 @@
+<!DOCTYPE html>
```

Comment:
nice template, might be worthwhile pointing folks to it as a new default example instead of lost in tests dir.

Reply:
this was just to resolve some issues with msg template lookup in tests, this is just a copy of the one found in msg-templates/default.html.j2

Reply:
nvm found a template that was already being used